taxi/forms.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number", )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    LENGTH_LICENSE = 8
+
+    class Meta:
+        model = Driver
+        fields = ("license_number", )
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) != DriverLicenseUpdateForm.LENGTH_LICENSE:
+            raise ValidationError(
+                "Length license number must be"
+                f" {DriverLicenseUpdateForm.LENGTH_LICENSE}"
+            )
+
+        if (
+                not license_number[:3].isalpha()
+                or (
+                license_number[:3] != license_number[:3].upper()
+                )
+        ):
```

Comment:
```suggestion
                or license_number[:3] != license_number[:3].upper()
        ):
```

Reply:
Could you provide this change? Or it is not match with line length?